config/nginx.conf.erb

```diff
@@ -81,6 +81,9 @@ http {
             uwsgi_param SERVER_PORT $server_port;
             uwsgi_param SERVER_NAME $server_name;
             uwsgi_pass unix:/tmp/nginx.socket;
+            uwsgi_buffer_size 16k;
```

Comment:
Is this just common settings which weren't added before, or is there a need specific to this PR?

Reply:
Yup, the SAML requests contain some rather long XML strings, so we needed to increase this to avoid nginx errors